src/LeftNav/right-arrow.svg

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
clients don't have to copy it now. webpack builds it into the component. do you still think it needs to be made into a component?